telemetry/definitions/commonDefinitions.json

```diff
@@ -282,6 +282,11 @@
                 "RemoteRunProject"
             ]
         },
+        {
```

Comment:
Question: Are you adding this new type to a metric? Will need to define that here as well.

Reply:
Oh shoot forgot to do that, fixing now